src/digital_identity_service/index.js

```diff
@@ -9,6 +13,45 @@ class DigitalIdentityService {
     this.pem = pem;
     this.apiUrl = apiUrl;
   }
+
+  /**
+   *
+   * @param shareSessionConfig
+   * @returns {Promise<ShareSessionResult>}
+   */
+  createSession(shareSessionConfig) {
+    const jsonShareSession = JSON.parse(JSON.stringify(shareSessionConfig));
+
+    const payload = new Payload(jsonShareSession);
+
+    const requestBuilder = new RequestBuilder()
+      .withBaseUrl(this.apiUrl)
+      .withHeader('X-Yoti-Auth-Id', this.sdkId)
+      .withPemString(this.pem)
+      .withEndpoint('/v2/sessions')
```

Comment:
`/v2` is not needed anymore since the apirUrl is encapsulating that. You can just have `/sessions` that will be enough.